reward/staking_manager.go

```diff
@@ -46,7 +46,7 @@ func newStakingManager(bc *blockchain.BlockChain, gh governanceHelper) *stakingM
 }
 
 // GetStakingInfoFromStakingCache returns a corresponding stakingInfo for a blockNum.
```

Comment:
```suggestion
// getStakingInfo returns a corresponding stakingInfo for a blockNum.
```

Reply:
I'll make a new pr for method description comments

Reply:
This is a typo. Let's fix it in the next PR.